src/java/org/apache/cassandra/io/sstable/SSTableSimpleWriter.java

```diff
@@ -36,64 +42,132 @@
  * means that rows should be added by increasing md5 of the row key. This is
  * rarely possible and SSTableSimpleUnsortedWriter should most of the time be
  * prefered.
+ * <p>
+ * Optionally, the writer can be configured with a max SSTable size for SSTables.
+ * The output will be a series of SSTables that do not exceed a specified size.
+ * By default, all sorted data are written into a single SSTable.
  */
 class SSTableSimpleWriter extends AbstractSSTableSimpleWriter
 {
+    private final long maxSSTableSize;
```

Comment:
can we either add a comment that mentions the units for the size, or better yet rename the variable to indicate it measures in bytes?
```suggestion
    private final long maxSSTableSizeInBytes;
```